crates/starknet-devnet-core/src/starknet/starknet_config.rs

```diff
@@ -128,6 +146,7 @@ pub struct StarknetConfig {
     pub strk_erc20_contract_class: String,
     #[serde(skip_serializing)]
     pub predeclare_argent: bool,
+    pub class_size_config: ClassSizeConfig,
```

Comment:
Would it make sense to avoid suffix `_config` here?

Reply:
It would, but we already have `fork_config`, so I'll just leave it like this.